flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobVertexMessageParameters.java

```diff
@@ -26,6 +26,8 @@
 public class JobVertexMessageParameters extends JobMessageParameters {
 
     public final JobVertexIdPathParameter jobVertexIdPathParameter = new JobVertexIdPathParameter();
+    public final FlameGraphTypeQueryParameter flameGraphTypeQueryParameter =
```

Comment:
Well this is pretty jank. Why is there not a separate parameters class extending this one?

Reply:
My bad! Support for different Flame Graphs type was the last minute edition and I misunderstood how those `JobVertexMessageParameters` are intended to be used. Fixing it.

Reply:
Already solved.